internal/command/user_human.go

```diff
@@ -179,6 +180,19 @@ func (c *Commands) createHuman(ctx context.Context, orgID string, human *domain.
 	if err := human.CheckOrgIAMPolicy(orgIAMPolicy); err != nil {
 		return nil, nil, err
 	}
+	if !orgIAMPolicy.UserLoginMustBeDomain {
```

Comment:
does a username always have to be an email address?

Reply:
won't we already always use email as username on register and add on these orgs?